modules/pubCommonIdSystem.js

```diff
@@ -7,11 +7,14 @@
 
 import * as utils from '../src/utils.js';
 import {submodule} from '../src/hook.js';
+import {getCoreStorageManager} from '../src/storageManager.js';
 
 const PUB_COMMON_ID = 'PublisherCommonId';
 
 const MODULE_NAME = 'pubCommonId';
 
+const coreStorage = getCoreStorageManager('userid');
```

Comment:
Please use this function https://github.com/prebid/Prebid.js/blob/master/src/storageManager.js#L305 to get storage functions
I have added hooks on getStorageManager function which will not allow device access depending on config